Makefile

```diff
@@ -11,47 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include Makefile.INCLUDE
+VERSION  := 0.1.0
```

Comment:
With alertmanager as an exception, I would go for 0.0.1 to point out to people just how experimental it is...